### DISCUSSION TITLE
Venu 2S software error

### DISCUSSION THREAD

**Discussion by Kaetzy** [maca88/SmartBikeLights#60] [General]
on the Venu 2s the data field is not displayed.
But when I start cycling it is not displayed, only when I stop and then click start again does the data field appear.
Is it possible to control the light via the data field?
When I click on the data field it doesn't work and I don't know anything else.

**Comment by maca88** [comment]
That is odd. Do you see an empty field or you see a text like: "No Network", "Error 3"?

> Is it possible to control the light via the data field?

You can manually control the light by using the data field menu by holding the "Up" button. You can check the details here:
https://github.com/maca88/SmartBikeLights?tab=readme-ov-file#changing-control-modes

**Comment by Kaetzy** [comment]
"No Network" is coming up. If you want a video about it, I'll make one.
Or can I help you in any other way?

**Reply by maca88** [reply]
"No Network" means that the light was not paired with your watch. Make sure that the light is paired, you should see the light under the Sensors menu. If you have a Varia, most likely only the radar was paired. You can check the following video on how to pair both the light and radar in case you have Varia: https://www.youtube.com/watch?v=G-lgdH2mZwg

**Comment by Kaetzy** [comment]
Hey,
how it is connected is clear. I made a video to make it easier to understand

https://www.youtube.com/shorts/P-NppQQZ6rI

**Reply by maca88** [reply]
Hi,
I tried to reproduce this issue on fenix 7 without success, on fenix it works the first time. On fenix, it takes approximate one second to switch from "No network" to display the light icon. Do you have any other data fields installed on Venu 2s? I am asking this because there are some cases where a different data field can interfere with Smart Bike Lights. One example is the "Tempe Visualizer" data field. When installed alongside Smart Bike Lights, the screen is not refreshed and you need to swipe up and down make it work.

**Comment by Kaetzy** [comment]
Sorry I'm only getting in touch now, I was a bit stressed.
I don't have any other date field installed. I only have other clock representations/display designs

**Comment by Kaetzy** [comment]
The manufacturer says it is not possible to control the light via the clock.
That's why it's not possible to control via the data field
